lib/commonBindings.js

```diff
@@ -55,7 +55,13 @@ function parseMessage(message) {
 
     try {
         stringMessage = message.toString();
-        parsedMessage = JSON.parse(stringMessage);
+        try {
+            parsedMessage = JSON.parse(stringMessage);
+        } catch (e) {
+            stringMessage = message.toString('hex');
+            parsedMessage = stringMessage;
+        }
+        config.getLogger().debug(context, 'stringMessage: [%s]', stringMessage);
```

Comment:
parsedMessage should be logged?

Reply:
done in 8d4ad5be96c186a678c4674be4e0f3e9cc63e2d0